src/workerd/server/workerd.c++

```diff
@@ -60,6 +60,11 @@
 #define environ (*_NSGetEnviron())
 #endif
 
+#include <workerd/util/perfetto-tracing.h>
+#if defined(WORKERD_USE_PERFETTO)
+PERFETTO_USE_CATEGORIES_FROM_NAMESPACE(workerd::traces);
+#endif
```

Comment:
I suspect we might want to collapse these four lines into just `#include <workerd/util/use-perfetto-categories.h>`